server/server.go

```diff
@@ -888,13 +892,15 @@ func (s *Server) Start() error {
 		s.ProjectCmdOutputHandler.Handle()
 	}()
 
-	server := &http.Server{Addr: fmt.Sprintf(":%d", s.Port), Handler: n}
+	tlsConfig := &tls.Config{GetCertificate: s.GetSSLCertificate}
```

Comment:
We use an older version of golang https://pkg.go.dev/crypto/tls@go1.17.13

Seems like crypto/tls defaults to 1.0 in the older version. The newer golang it defaults to 1.2.

```go
 	// MinVersion contains the minimum TLS version that is acceptable.
	// If zero, TLS 1.0 is currently taken as the minimum.
	MinVersion uint16
```

So we should explicitly set it here to 1.2 for now.

```suggestion
	tlsConfig := &tls.Config{GetCertificate: s.GetSSLCertificate, MinVersion: 1.2}
```

Reply:
@camillsir unsure if this is the correct syntax. Please double check me

Reply:
Ah, yes. I think I set the correct value now.